katas/content/single_qubit_gates/Common.qs

```diff
@@ -0,0 +1,6 @@
+namespace Kata.Verification {
+
+    operation ApplyToFirstCA(op : Qubit => Unit is Adj + Ctl, qs : Qubit[]) : Unit is Adj + Ctl {
```

Comment:
I think once you update the testing harness for basis_change to use the same code as others, you shouldn't need this one. Generally, one-liner operations like this can be defined using lambda expressions: `let solution = register => Kata.SignFlip(register[0]);` defines solution as an operation that applies `Kata.SignFlip` to the first element of `register`, which is the same as this operation does!